src/main/java/TaskToEvent.java

```diff
@@ -1,2 +1,5 @@
-package main.java;public interface TaskToEvent {
+package main.java;
+
+public interface TaskToEvent {
+    public Event createEventFromTask(Task task, Schedule schedule)
```

Comment:
Dependency Inversion. Make sure we're not making abstract things rely on concrete things (this will depend on whether Task ends up being abstract or not)